test/lib/navigation.rb

```diff
@@ -0,0 +1,18 @@
+require 'page_navigation'
+
+module OllertTest
+  module Navigation
+    include PageNavigation
+
+    def on(cls, &block)
+      screen = cls.new
+      block.call screen if block
```

Comment:
6 of 1 1/2 dozen of another :) Either way is fine with me. Would it be better if I said `if block_given?` instead?